cmd/beaker/session.go

```diff
@@ -67,17 +67,17 @@ To pass flags, use "--" e.g. "create -- ls -l"`,
 		Args: cobra.ArbitraryArgs,
 	}
 
-	var altHome bool
+	var localHome bool
 	var image string
 	var name string
 	var node string
 	var pull string
-	cmd.Flags().BoolVar(&altHome, "alt-home", false, "Mount alternate home directory managed by Beaker (experimental)")
 	cmd.Flags().StringVar(
 		&image,
 		"image",
 		"beaker://ai2/cuda11.2-ubuntu20.04",
 		"Base image to run, may be a Beaker or Docker image")
+	cmd.Flags().BoolVar(&localHome, "local-home", false, "Ignore Beaker configuration and always mount the invoking user's home directory")
```

Comment:
```suggestion
	cmd.Flags().BoolVar(&localHome, "local-home", false, "Mount the invoking user's home directory, ignoring Beaker configuration for this")
```

"always" confused me for a moment.